src/screens/MemberDetail/MemberDetail.tsx

```
@@ -253,11 +257,76 @@ const MemberDetail: React.FC<MemberDetailProps> = ({ id }): JSX.Element => {
+        style={{
+          display: 'flex',
+          marginTop: '2rem',
+          width: 'max',
+          padding: '2rem',
+        }}
+        className={`bg-white gap-4 ${styles.allRound}`}
+      >
+        <a href="/member">
+          <button
+            className={`${styles.featuresBox} ${styles.allRound} flex items-center rounded-lg text-gray-100 transition-colors`}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)